components/user_group_popover/group_member_list/group_member_list.tsx

```diff
@@ -170,7 +170,10 @@ const GroupMemberList = (props: Props) => {
                     key={user.id}
                     role='listitem'
                 >
-                    <UserButton onClick={() => showUserOverlay(user)}>
+                    <UserButton
+                        onClick={() => showUserOverlay(user)}
+                        aria-haspopup={true}
```

Comment:
Should this be `dialog` since it opens a popover?

Reply:
That's right. Thanks for catching that.